images/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM golang:alpine3.12 as builder
+FROM golang:1.17-alpine as builder
 
 COPY . /usr/src/sriov-network-device-plugin
 ADD images/ddptool-1.0.1.12.tar.gz /tmp/ddptool/
 
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
-RUN apk add --no-cache --virtual build-dependencies build-base linux-headers
+RUN apk add --no-cache --virtual build-dependencies build-base linux-headers git
```

Comment:
why do we need git in a builder image?

Reply:
The new version of alpine does not include git anymore and golang requires git to pull dependencies. I also needed to add git on my setup when I was testing this PR.